app/views/notes/_note.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="card">
+  <h5 class="card-header">Created at: <%= note.created_at.to_date.strftime("%d %b, %Y") %></h5>
```

Comment:
make a helper method for this `application_helper.rb`
```ruby
def formatted_date(datetime)
  datetime.to_date.strftime("%d %b, %Y")
end
```

and use here.

Reply:
done